src/bitmessagecli.py

```diff
@@ -354,8 +390,8 @@ def bmSettings(): #Allows the viewing and modification of keys.dat settings.
             else:
                 print "\n     Invalid input. Please try again.\n"
                 invalidInput = True
-                
-            if invalidInput != True: #don't prompt if they made a mistake. 
+
+            if invalidInput is not True:  # don't prompt if they made a mistake.
```

Comment:
if not invalidInput:

Reply:
I can't go that far, generally speaking. `invalidInput` might be truthy without being True or falsey without being False, thus changing the meaning of the code. You may be right that we can do this here but it is out of scope for this work.